client/fs_endpoint.go

```diff
@@ -395,13 +396,6 @@ func (f *FileSystem) logs(conn io.ReadWriteCloser) {
 		return
 	}
 
-	state, ok := alloc.TaskStates[req.Task]
```

Comment:
Pretty unfortunate to loose this. Why can't we inspect the start time for the task?

Reply:
We lost it because `AR.Alloc()` is no longer mutated on task state updates. It represents the immutable server-sent-view of the alloc.

I can add the ability for the Client to inspect TaskStates back in, but it was just nice not to have to plumb that all the way through for an improved error message.